packages/@expo/cli/src/start/server/UrlCreator.ts

```diff
@@ -60,7 +60,10 @@ export class UrlCreator {
       return null;
     }
 
-    const manifestUrl = this.constructUrl({ ...options, scheme: 'http' });
+    const manifestUrl = this.constructUrl({
+      ...options,
+      scheme: this.defaults?.hostType === 'tunnel' ? 'https' : 'http',
```

Comment:
Likely want to toggle this to be https by default https://github.com/expo/expo/blob/b5d471d6d5b117a58ff129a3752e5bec2af08c7b/packages/%40expo/cli/src/start/server/UrlCreator.ts#L95 if all tunnel URLs should be https. Don't rely on defaults, use `options.hostType`

Reply:
The problem is that most of the time `options` is an empty object